Makefile

```diff
@@ -203,7 +203,7 @@ ifneq (${CLANG_FORMAT_EXCLUDES},)
 endif
 
 ifneq (${BUILD_MISSING_DEPS},)
-  MY_CMAKE_FLAGS += -DBUILD_MISSING_DEPS:BOOL=${BUILD_MISSING_DEPS}
+  MY_CMAKE_FLAGS += -DOpenImageIO_BUILD_MISSING_DEPS:STRING=${BUILD_MISSING_DEPS}
 endif
```

Comment:
Actually, I'd recommend removing these 3 lines entirely. Let's just document that people use `OpenImageIO_BUILD_MISSING_DEPS`. Because that variable is set up in dependency_util.cmake using a call to set_cache, it will automatically pick up a value in an environment variable, which is naturally set up by make. In other words

    make OpenImageIO_BUILD_MISSING_DEPS=all

will just work, even if Makefile doesn't mention it at all.

Reply:
set_cache is a recent addition, but I'm really liking it, so my goal is to slowly change lots more things that are `set(... CACHE)` to set_cache (and `option` to `set_option`) and then as we do so, eliminate any special processing of those variables in the Makefile. My goal is to eventually get this Makefile really down the the minimum wrapper needed to have `make` give you a decent default build without needing to know anything about cmake.